packages/editor/src/components/media-placeholder/index.native.js

```diff
@@ -1,27 +1,26 @@
 /**
  * External dependencies
  */
-import { View, Text, Button } from 'react-native';
+import { View, Text, TouchableWithoutFeedback } from 'react-native';
 
 import styles from './styles.scss';
 
 import { __ } from '@wordpress/i18n';
+import { Dashicon } from '@wordpress/components';
 
 function MediaPlaceholder( props ) {
 	return (
-		<View style={ styles.emptyStateContainer }>
-			<Text style={ styles.emptyStateTitle }>
-				{ __( 'Image' ) }
-			</Text>
-			<Text style={ styles.emptyStateDescription }>
-				{ __( 'Upload a new image or select a file from your library.' ) }
-			</Text>
-			<View style={ styles.emptyStateButtonsContainer }>
-				<Button title={ __( 'Device Library' ) } onPress={ props.onUploadMediaPressed } />
-				<Button title={ __( 'Take photo' ) } onPress={ props.onCapturePhotoPressed } />
+		<TouchableWithoutFeedback onPress={ props.onMediaOptionsPressed }>
+			<View style={ styles.emptyStateContainer }>
+				<Dashicon icon={ 'cover-image' } />
+				<Text style={ styles.emptyStateTitle }>
+					{ __( 'Image' ) }
+				</Text>
+				<Text style={ styles.emptyStateDescription }>
+					{ __( 'UPLOAD IMAGE' ) }
```

Comment:
Maybe change this string to: "Select Image" ?

Reply:
Hey @SergioEstevao , thanks for the comment.
You are right. I will fix that.